vlib/x/ttf/README.md

```diff
@@ -210,7 +210,7 @@ It is possible to modify these parameters to obtain the desired effect on the te
 ## TTF Sokol render
 The sokol render use the  bitmap render to create the text and the `gg` functions to render
 the text to the screen.
-It is mor esimpel to use in a `gg app` that the raw bitmap render.
+It is more simple to use in a `gg app` that the raw bitmap render.
```

Comment:
`simpler` is more correct. `that` typo as well.

Reply:
Fixed in 160a887.